packages/fast-components-react-msft/src/action-toggle/README.md

```diff
@@ -0,0 +1,13 @@
+# Action Toggle
+*Action toggle* should be used as control on or around an interactive region of the page that affects the region or items within the region. It usually represents a switch between one of two states.
+
+## Usage
+Use *action toggle* for specific control around a functional state. Labeling can be done using a glyph and text, but could also contain just a glyph or just text. Make sure the text and glyph clearly represent the expected function. When only a glyph is used to label the button, include a tooltip in a span element following the button element.
+
+The "selectedGlyph" and "unselectedGlyph" props take a function that returns a ReactNode to render the glyph (ie. a [render prop](https://reactjs.org/docs/render-props.html)).  Note that the function should accept a string parameter that is applied as a classname to the top level element of the resulting node.  This classname  will be populated by the *action toggle* component to use an internally generated class ("actionToggle_selectedGlyph" & "actionToggle_unselectedGlyph") to apply to the glyphs for styling. 
+
+## Style guidance
+Multiple related *action toggle* controls should be grouped together visually to reinforce their associated functionality. Place the toggle where it is not hidden or obscured in the clutter of the page.
+
+## Accessibility
+The *action toggle* utilizes the text found in the "selectedARIALabel" and "unselectedARIALabel" properties to populate an aria-label with the toggled state. This is important when the *action toggle* is configured as glyph-only. Otherwise, a non-sighted user would not have context to the toggle change. If a more verbose description is needed, tooltip is always an option to provide additional context.
```

Comment:
Same thing about tooltip here too

Reply:
"The aria-label is important when ..."

Reply:
"Otherwise, a non-sighted user would not get the context of the toggle change."

Reply:
If the tooltip sentence stays: "If a more verbose description is needed, a tooltip is an option to provide additional context."

Reply:
Applied Jason's edits.  Same notion about tooltip as above, seems like unnecessary churn.

Reply:
I like keeping the tooltip so long as we are not referencing it as a component. It's a nit, but I would suggest ensuring that we call it out as "a tooltip" so that we aren't referencing something internal ("tooltip" as a thing we have), but rather the accessible pattern/control in general.

Reply:
Yep, tooltips exist even if we don't have a component yet.  How about "If a more verbose description is needed, a tooltip is an option to provide additional context."?

Reply:
@scomea yep - exactly. I think the only thing missing from the last sentance above is "a" to denote, "a tooltip". Agree with this 100%